dspace-api/src/test/java/org/dspace/authorize/AuthorizeServiceTest.java

```diff
@@ -125,4 +128,39 @@ public void testauthorizeMethodRespectSpecialGroups() {
             throw new AssertionError(ex);
         }
     }
+
+    @Test
+    public void testIsCollectionAdmin() throws SQLException, AuthorizeException {
+
+        context.turnOffAuthorisationSystem();
+
+        Community community = communityService.create(null, context);
+        Collection collection = collectionService.create(context, community);
+        EPerson eperson = ePersonService.create(context);
+
+        Group administrators = collectionService.createAdministrators(context, collection);
+        groupService.addMember(context, administrators, eperson);
+
+        context.restoreAuthSystemState();
+        context.commit();
+
+        Assert.assertTrue(authorizeService.isCollectionAdmin(context, eperson));
+    }
+
+    @Test
+    public void testIsCollectionAdminReturnsTrueIfTheUserIsCommunityAdmin() throws SQLException, AuthorizeException {
+
+        context.turnOffAuthorisationSystem();
+
+        Community community = communityService.create(null, context);
+        EPerson eperson = ePersonService.create(context);
+
+        Group administrators = communityService.createAdministrators(context, community);
+        groupService.addMember(context, administrators, eperson);
```

Comment:
Same here, use Builders please.

Also, for full test completion, would you be willing to add a `testIsCommunityAdmin()` method here...as it seems to be missing & it'd be good to also prove that a Community Admin also results in that method returning `true`, while a Collection Admin results in that method returning `false`.